tests/testsuite/check_cfg.rs

```diff
@@ -53,7 +53,7 @@ fn features() {
         .build();
 
     p.cargo("build -v -Zcheck-cfg=features")
-        .masquerade_as_nightly_cargo()
+        .masquerade_as_nightly_cargo(&["check-cfg"])
```

Comment:
Haven't dealt with this one before, unsure if we should do it as `check-cfg` or `check-cfg=features`

Reply:
I think `check-cfg` is better since that is that the [unstable feature is called](https://github.com/rust-lang/cargo/blob/d583b21e1494268be2cc8c19950e94f03ae3bce2/src/doc/src/reference/unstable.md#check-cfg). If certain parts get stabilized before others (like [`features`](https://github.com/rust-lang/cargo/blob/8529eb1c9641a75667d1fccd8025c3fe46c803d7/src/cargo/core/features.rs#L905-L908)), at that point it should become `check-cfg=features`